plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -52,8 +52,8 @@ func getAllTolerationPreferNoSchedule(tolerations []v1.Toleration) (tolerationLi
 	return
 }
 
-func getTolerationListFromPod(pod *v1.Pod) ([]v1.Toleration, error) {
```

Comment:
Makes sense @aveshagarwal and @bsalamat - Initially I wanted to do the same but since the issue was related to eliminating deadcode, I didnot touch that. If you guys are ok with including the change, I can do that here itself.